webserver/views/api.py

```diff
@@ -140,7 +142,10 @@ def _validate_auth_header():
 
     user = db.user.get_by_token(auth_token)
     if user is None:
-        user = db.user.create_user_with_token(auth_token)
+        if CREATE_FAKE_USERS_FOR_STRESS_TESTING:
```

Comment:
I know this has been merged, but is there a way that this can be integrated without having a create operation in a verify method?
Is it possible to make the stress-test tool send 2 requests (1 POST to create an account [under a similar guard as this one], and then the POST to save data)?
Is this setting used when `DEBUG` is set to `False`? If no, we could have a guard to make sure that `CREATE_FAKE_USERS_FOR_STRESS_TESTING && !DEBUG` is never true